dockerfiles/centos-stream8-vdbench5.04.07 pod/Dockerfile

```diff
@@ -0,0 +1,46 @@
+FROM quay.io/centos/centos:stream8
+# copying vdbench
+COPY vdbench50407.zip  /vdbench/vdbench50407.zip
+COPY vdbench_runner.sh /vdbench/vdbench_runner.sh
+
+# installing java
+RUN yum -y install java
+
+# installing vim
+RUN yum -y install vim
```

Comment:
Why?

Reply:
@bbenshab  add in order to debug it simple in case of, right Boaz ?

Reply:
Yeah, I hate that but I can't say that I haven't done that.